upgrade_project/docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+networks:
+  http_net:
+  db_net:
+  es_net:
+
+volumes:
+  cms-shared:
+  dbdata:
+  esdata:
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.9.1
+    environment:
+      - cluster.name=elastic-cluster
+      - discovery.type=single-node
+      - data
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xmx1G "
+    #ports:
+      #- 9200:9200
+      #- 9600:9600
+    volumes:
+      - esdata:/usr/share/elasticsearch/data
+    networks:
+      - es_net
+
+  dotcms:
+    image: ${docker_tag}
+    environment:
+      "CATALINA_OPTS": ' -Xms1g -Xmx1g '
+      "DB_BASE_URL": "jdbc:postgresql://db/dotcms"
+      "DB_USERNAME": 'dotcmsdbuser'
+      "DB_PASSWORD": 'password'
+      "DOT_ES_AUTH_BASIC_PASSWORD": 'admin'
+      "DOT_INITIAL_ADMIN_PASSWORD": "admin"
+      "DOT_ES_ENDPOINTS": 'http://elasticsearch:9200'
+      "CUSTOM_STARTER_URL": ${custom_starter}
+  
+      "DOT_CIRCUIT_BREAKER_MAX_CONN_TOTAL": "3"
+      
+      #EDIT CONTENT
+      "DOT_CONTENT_EDITOR2_ENABLED": "true"
+      "DOT_CONTENT_EDITOR2_CONTENT_TYPE" : "true"
+      "DOT_FEATURE_FLAG_NEW_EDIT_PAGE": "true"
+
+    depends_on:
+      - elasticsearch
+      - db
+    volumes:
+      - cms-shared:/data/shared
+      #- [serverpath]/license.zip:/data/shared/assets/license.zip
+    networks:
+      - db_net
+      - es_net
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+
+  db:
+    image: postgres:15
```

Comment:
use `image: pgvector/pgvector:pg16`

Reply:
Unable to use this since the version that we are trying to upgrade is not compatible with this pg version (dotcms/dotcms:23.10.24_lts_v17_8b0be94)